data/data/agent/files/usr/local/bin/agent-auth-token-status.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+
+# shellcheck disable=SC1091
+source "issue_status.sh"
+
+status_issue="65_token"
+
+export TZ=UTC
+check_token_expiry() {
+  expiry_epoch=$(date -d "${AGENT_AUTH_TOKEN_EXPIRY}" +%s)
+  current_epoch=$(date +%s)
+
+  if [ "$current_epoch" -gt "$expiry_epoch" ]; then
+    printf '\\e{lightred}The authentication token has expired. Please re-run the `openshift-install agent add-nodes` command to generate a new ISO, then reboot the node.\\e{reset}'| set_issue "${status_issue}"
```

Comment:
The assumption here is users generated the node iso with agent command. Should we also update this error message to guide users what they need to do if they had created the node iso using `oc`